.github/workflows/cflite.yaml

```diff
@@ -0,0 +1,26 @@
+name: ClusterFuzzLite PR fuzzing
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '**'
+permissions: read-all
+jobs:
+  PR:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address]
+    steps:
+    - name: Build Fuzzers (${{ matrix.sanitizer }})
+      id: build
+      uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+      with:
+        sanitizer: ${{ matrix.sanitizer }}
+        language: go
+    - name: Run Fuzzers (${{ matrix.sanitizer }})
+      id: run
+      uses: google/clusterfuzzlite/actions/run_fuzzers@v1
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does this need GITHUB_TOKEN?

If we keep this, can we limit its permissions to the needed subset? Hopefully this doesn't need more than `contents: read`.

Reply:
I am not sure _why_ ClusterfuzzLite needs the Github token, just that it is a requirement. 

[This discussion](https://github.com/google/clusterfuzzlite/issues/86) should cover the permissions required.

Reply:
This claims that `GITHUB_TOKEN` is `required:false`: https://github.com/google/oss-fuzz/blob/129f8f2639015d11ee2efa23c7ea83c6818b1f55/infra/cifuzz/actions/run_fuzzers/action.yml#L38

Does the workflow pass without it? Or is it possible I'm looking at the wrong workflow?